generators/bdk/python/templates/python/main.py.ejs

```diff
@@ -0,0 +1,46 @@
+import asyncio
```

Comment:
should we add a #!/usr/bin/env python3 at the beginning and make it executable? (I don't if it is common to do so)

Reply:
I am not sure about this shebang line because I always run `python3 main.py`